### DISCUSSION TITLE
Biggest pc case fan stopped working! Help!

### DISCUSSION THREAD

**Discussion by DmytroUKR82** [Rem0o/FanControl.Releases#2678] [Q&A]
Hi! I was trying to control my fans etc. so I installed latest version of fan control. It detected as rpm only gpu fans, cpu and one more (idk which one). I have 4 pre-installed fans (deepcool cg560)  - 3 front and one 140mm rear. I pressed on fan that was prm detected and it said something like -"make it to 0 to so you can find out which fan stop spinning". I did and after that my rear fan stopped spinning no matter what I do. I tried ti uninstall fan control with revo uninstaller so meaining all files etc. I changed inbios safe deafaults but nothing helps! It just drives me crazy! Please, help!

**Comment by Rem0o** [comment]
Have you tried to set it back to 100%?  Big fans have big inertia, and will need quite some power to start spinning.
